tools/webpack-helpers.ts

```diff
@@ -0,0 +1,72 @@
+import { basename } from 'path';
+
+/* Return imports-loader format to the config
+    For example:
+    [
+        // Adds 'imports-loader?this=>widndow'
+        {path: './foler/my_module.js', args: '?this=>window'},
+    ]
+*/
+interface importLoaderOptions {
+   path: string;
+   args: string;
+}
+function getImportLoaders (optionsArr:Array<importLoaderOptions>) {
+    let importsLoaders = [];
+    for(var loaderEntry of optionsArr) {
+        importsLoaders.push({
+            test: require.resolve(loaderEntry.path),
+            use: "imports-loader?" + loaderEntry.args
```

Comment:
Use template string here `imports-loader?${loaderEntry.args}`

Reply:
Not sure why the code needs to be more complicated than this.

Reply:
As i said, this is mostly nit-picking, as i read through this is what popped up in my mind, feel free to ignore.